pyaedt/hfss3dlayout.py

```diff
@@ -1400,21 +1400,21 @@ def import_awr(
     def import_ipc2581(
         self, input_file, aedb_path=None, control_file=None, set_as_active=True, close_active_project=False
     ):
-        """Import an IPC2581 file into HFSS 3D Layout and assign the stackup from an XML file if present.
+        """Import an ipc2581 file into HFSS 3D Layout and assign the stackup from an XML file if present.
```

Comment:
Again, this should be IPC2581 as originally written here and in subsequent lines.